src/olympia/amo/tests/__init__.py

```diff
@@ -90,6 +90,9 @@
 SQUOTE_ESCAPED = escape("'")
 # Unfortunately `escape` returns `&quot;` rather than the code so workaround:
 DQUOTE_ESCAPED = '&#x22;' if 'x' in SQUOTE_ESCAPED else '&#34;'
+# django3.2 uses fancy double quotes in its error strings
+STRING_QUOTE_OPEN = '“' if 'x' in SQUOTE_ESCAPED else "'"
+STRING_QUOTE_CLOSE = '”' if 'x' in SQUOTE_ESCAPED else "'"
```

Comment:
IMHO those should use a check on the django version, not on the value of `SQUOTE_ESCAPED` which has nothing to do with it.

Reply:
yeah I ended up doing that in the subsequent fix prs like #16481

Reply:
https://github.com/mozilla/addons-server/pull/16469/commits/0c543ce57ea30c08052d879b3122252f6e737b07